osm2gtfs/creators/trips_creator.py

```diff
@@ -13,32 +19,304 @@ def __repr__(self):
+                gtfs_trip.bikes_allowed = ""
+                gtfs_trip.direction_id = ""
+
+            # Calculate all times of stops, which were added with no time
+            Helper.interpolate_stop_times(gtfs_trip)
+        return trips_count
+
+    def _create_gtfs_service_period(self, feed, service):
+        """
+        Generate a transitfeed ServicePeriod object
+        from a time string according to the standard schedule:
+        https://github.com/grote/osm2gtfs/wiki/Schedule
+
+        :return: ServicePeriod object
+        """
+        try:
+            gtfs_service = feed.GetServicePeriod(service)
+            if gtfs_service is not None:
+                return gtfs_service
+        except KeyError:
+            pass
+
+        if service == "Mo-Fr":
+            gtfs_service = ServicePeriod(service)
+            gtfs_service.SetWeekdayService(True)
+            gtfs_service.SetWeekendService(False)
+        elif service == "Mo-Sa":
+            gtfs_service = ServicePeriod(service)
+            gtfs_service.SetWeekdayService(True)
+            gtfs_service.SetWeekendService(False)
+            gtfs_service.SetDayOfWeekHasService(5, True)
+        elif service == "Mo-Su":
+            gtfs_service = ServicePeriod(service)
+            gtfs_service.SetWeekdayService(True)
+            gtfs_service.SetWeekendService(True)
+        elif service == "Sa":
+            gtfs_service = ServicePeriod(service)
+            gtfs_service.SetWeekdayService(False)
+            gtfs_service.SetWeekendService(False)
+            gtfs_service.SetDayOfWeekHasService(5, True)
+        elif service == "Su":
+            gtfs_service = ServicePeriod(service)
+            gtfs_service.SetWeekdayService(False)
+            gtfs_service.SetWeekendService(False)
+            gtfs_service.SetDayOfWeekHasService(6, True)
+        elif service == "Sa-Su":
+            gtfs_service = ServicePeriod(service)
+            gtfs_service.SetWeekdayService(False)
+            gtfs_service.SetWeekendService(True)
+        elif re.search(r'^([0-9]{4})-?(1[0-2]|0[1-9])-?(3[01]|0[1-9]|[12][0-9])$', service):
+            service = service.replace('-', '')
+            gtfs_service = ServicePeriod(service)
+            gtfs_service.SetDateHasService(service)
+        else:
+            raise KeyError("Unknown service keyword: " + service)
+
+        gtfs_service.SetStartDate(self.config['feed_info']['start_date'])
+        gtfs_service.SetEndDate(self.config['feed_info']['end_date'])
+        feed.AddServicePeriodObject(gtfs_service)
+        return feed.GetServicePeriod(service)
+
+    def _load_itinerary_schedule(self, schedule, itinerary, service):
+        """
+        Load the part of the provided schedule that fits to a particular
+        itinerary.
+
+        :return times: List of strings
+        """
+        times = None
+        for trip in schedule['lines'][itinerary.route_id]:
+            trip_services = trip["services"]
+            if (trip[
```

Comment:
Odd place to break the line :thinking: Why not ?
```
if (trip['from'] == itinerary.fr 
           and trip['to'] == itinerary.to 
            and service in trip_services):
```

Reply:
Because my linter complains. Is it just mine? or I can do as you say? Which would be much nicer, indeed!

Reply:
Just checked here it [fails, too](https://travis-ci.org/grote/osm2gtfs/builds/317477394?utm_source=github_status&utm_medium=notification):

> osm2gtfs/creators/trips_creator.py:319:17: E129 visually indented line with same indent as next logical line

Please help me to produce a nicer readable line that sticks to our coding standards. This was the only one that worked for me, without breaking the line length.

Reply:
This one seems ok : 
```
            if (trip["from"] == itinerary.fr and
                    trip["to"] == itinerary.to and
                    service in trip_services):
                times = trip["times"]
```

Reply:
hey, yes @nlehuby this works and is much better. Thanks!